starboard/elf_loader/exported_symbols.cc

```diff
@@ -632,7 +632,9 @@ const void* ExportedSymbols::Lookup(const char* name) {
   const void* address = map_[name];
   // Any symbol that is not registered as part of the Starboard API in the
   // constructor of this class is a leak, and is an error.
-  SB_CHECK(address) << "Failed to retrieve the address of '" << name << "'.";
+  if (!address) {
```

Comment:
Consider adding a function comment in the header to document that the address will be NULL if the symbol isn't found.